lua/apisix/plugins/wolf-rbac.lua

```diff
@@ -77,6 +81,18 @@ local function create_rbac_token(appid, wolf_token)
     return token_version .. "#" .. appid .. "#" .. wolf_token
 end
 
+local function fail_response(message, init_values)
+    local response = init_values or {}
+    response.message = message
+    return response
+end
+
+local function success_response(message, init_values)
+    local response = init_values or {}
+    response.message = message
```

Comment:
how about this name `error_msg` ?

Reply:
I see that the field in `jwt-auch` is called "message". But the core module uses "error_msg". I chose to be consistent with `jwt-auch`

In the aspect of returning error information, I think the core module should provide a unified function to handle it. In this way, the structure of error/correct information returned by the whole system can be consistent. This is still important for API services. Otherwise, the client may have to handle several different information structures